packages/contracts/src/RelayHub.sol

```diff
@@ -381,6 +386,15 @@ contract RelayHub is IRelayHub, Ownable {
         }
     }
 
+    function calculateDevCharge(uint256 charge) public view returns (uint256){
+        if (config.devFee == 0){ // save some gas in case of zero dev charge
+            return 0;
+        }
+        unchecked {
```

Comment:
Is it only for gas optimisations? If so, how much gas do we save doing it?

Reply:
It saves a few hundreds, and it's safe.